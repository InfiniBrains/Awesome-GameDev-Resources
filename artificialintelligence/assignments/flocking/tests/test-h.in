1.000 2.000 1.000 0.000 1.000 2.000 0.000 2
0.000 0.500 -2.000 -1.000
0.000 -0.500 2.000 3.000
0.125
0.125
