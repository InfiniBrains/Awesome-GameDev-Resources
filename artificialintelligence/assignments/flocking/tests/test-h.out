-0.250 0.391 -2.000 -0.875
0.250 -0.141 2.000 2.875
-0.514 0.295 -2.114 -0.765
0.514 0.205 2.114 2.765
